python/tvm/relay/frontend/tflite.py

```diff
@@ -1872,7 +1872,7 @@ def convert_fully_connected(self, op):
                 out_dtype="int32",
             )
         else:
-            out = _op.nn.dense(in_expr, weight_expr)
+            out = _op.nn.dense(in_expr, weight_expr, units=weight_shape[0])
```

Comment:
One small suggestion, if possible, can we add a test case which will indicate the issue fixed here. So that it will help in future breaks.